COCOAPODS.md

```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
+In the configuration block specify the C interoperability options:
+
+    * `extraOpts` – to specify the list of options to a Pod library. For example, specific flags
+    * `packageName` – to specify the package name. If you specify it, you can import the library using the package name: `import <packageName>`
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+   
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("new_pod") {
+               extraOpts = listOf("-compiler-option")
+               packageName = "package"
+           }
+
+       }
+   }
+   ```
+
+   </div>
+
+3. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+If you use the `packageName` parameter, you can import the library using the package name: `import <packageName>`:
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import packageName
+```
+
+</div>
+
+### Add a dependency on a static Pod library
+
+You can add dependencies on a static Pod library with `pod()` and `useLibraries()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of the library using the `pod()` function.
+
+2. Call the `useLibraries()` function: it enables a special flag for static libraries.
+
+3. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("YandexMapKit") {
+               version = "~> 3.2"
+           }
+           useLibraries()
+
+       }
+   }
+   ```
+
+   </div>
+
+
+4. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+### Update Podfile for Xcode
+
+If you already have the Xcode project, CocoaPods plugin requires Podfile changes to work correctly:
```

Comment:
> If you already have the Xcode project

Seems awkward. What if I don't have the Xcode project, but going to create one?
What if I do, but not going to use it?

This should probably be more like "If you would like to import your Kotlin project to an Xcode project,".